templates/__common__/config/tasks/graphics-meta.js

```diff
@@ -0,0 +1,362 @@
+// native
+const path = require('path');
+
+// packages
+const colors = require('ansi-colors');
+const fs = require('fs-extra');
+const glob = require('fast-glob');
+const puppeteer = require('puppeteer');
+const quaff = require('quaff');
+
+// internal
+const paths = require('../paths');
+const config = require('../../project.config');
+const { ensureSlash, logErrorMessage } = require('../utils');
+
+// used for screenshots
+const TABLET_BREAKPOINT = process.env.TABLET_BREAKPOINT || 768;
+const MOBILE_BREAKPOINT = process.env.MOBILE_BREAKPOINT || 475;
+const viewportOpts = size => {
+  return {
+    width: Number(size === 'tablet' ? TABLET_BREAKPOINT : MOBILE_BREAKPOINT),
+    height: 1000,
+    deviceScaleFactor: 2,
+  };
+};
+
+// path where chrome is typically installed on MacOS
+const CHROME_INSTALL_PATH =
```

Comment:
perhaps! In my mind, it'd be pretty consistent for us since we're all on macs, but that is probably very short-sighted/OS discriminating of me 😆. Do you think it's a blocker? I do see [solves](https://github.com/hughsk/chrome-location/blob/master/index.js) out there in the world we could investigate

Reply:
Hahah I saw similar npm packages! Could we just steal their raw code lololol, I was hesitant to add another package. I think I'm just wary about new Mac updates, and whether that could change paths on the machine, but maybe that's not very likely?

Reply:
Can confirm that `chrome-location` does find the correct location: `/Applications/Google Chrome.app/Contents/MacOS/Google Chrome`

Reply:
... Does anyone at the Tribune use a Windows or did we just shame all of them into using Mac's

Reply:
I'm in favor of making it more OS aware in a new PR since this one has grown to be so giant 😅